docs/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.md

```diff
@@ -28,6 +29,279 @@
+		* Removed property: `aids_risk_factors`
+		* Removed property: `bmi`
+		* Removed property: `body_surface_area`
+		* Removed property: `cd4_count`
+		* Removed property: `cdc_hiv_risk_factors`
+		* Removed property: `comorbidities`
+		* Removed property: `comorbidity`
+		* Removed property: `comorbidity_method_of_diagnosis`
+		* Removed property: `days_to_comorbidity`
+		* Removed property: `days_to_risk_factor`
+		* Removed property: `diabetes_treatment_type`
+		* Removed property: `dlco_ref_predictive_percent`
+		* Removed property: `eye_color`
+		* Removed property: `fev1_fvc_post_bronch_percent`
+		* Removed property: `fev1_fvc_pre_bronch_percent`
+		* Removed property: `fev1_ref_post_bronch_percent`
+		* Removed property: `fev1_ref_pre_bronch_percent`
+		* Removed property: `haart_treatment_indicator`
+		* Removed property: `height`
+		* Removed property: `hepatitis_sustained_virological_response`
+		* Removed property: `hiv_viral_load`
+		* Removed property: `hormonal_contraceptive_type`
+		* Removed property: `hormonal_contraceptive_use`
+		* Removed property: `hpv_positive_type`
+		* Removed property: `hysterectomy_margins_involved`
+		* Removed property: `hysterectomy_type`
+		* Removed property: `immunosuppressive_treatment_type`
+		* Removed property: `menopause_status`
+		* Removed property: `nadir_cd4_count`
+		* Removed property: `pancreatitis_onset_year`
+		* Removed property: `pregnancy_count`
+		* Removed property: `pregnancy_outcome`
+		* Removed property: `reflux_treatment_type`
+		* Removed property: `risk_factor`
+		* Removed property: `risk_factor_method_of_diagnosis`
+		* Removed property: `risk_factor_treatment`
+		* Removed property: `risk_factors`
+		* Removed property: `undescended_testis_corrected`
+		* Removed property: `undescended_testis_corrected_age`
+		* Removed property: `undescended_testis_corrected_laterality`
+		* Removed property: `undescended_testis_corrected_method`
+		* Removed property: `undescended_testis_history`
+		* Removed property: `undescended_testis_history_laterality`
+		* Removed property: `viral_hepatitis_serologies`
+		* Removed property: `weight`
+		* Changes made to `days_to_first_event`
+		* Changes made to `timepoint_category`
+			* New permissable value: `After Study Registration`
+			* New property: `deprecated_enum`
+* Altered `read_group` Entity
+	* Changes made to `deprecated`
+	* Removed permissable value: `RIN`
+	* Changes made to `properties`
+		* Changes made to `single_cell_library`
+			* New permissable value: `Chromium 3' Gene Expression v1 Library`
+			* New permissable value: `Chromium 3' Gene Expression v4 Library`
+			* New permissable value: `Chromium Next GEM Single Cell Multiome ATAC + Gene Expression Library`
+		* Changes made to `target_capture_kit`
+			* New permissable value: `xGen Exome Research Panel v2.0`
+* Altered `sample` Entity
+	* Changes made to `deprecated`
+	* New permissable value: `tumor_code_id`
+	* Removed permissable value: `composition`
```

Comment:
Script:  I'm not sure why it thinks these properties are permissible values.  But it looks like those are accounted for below, so we can probably remove these lines.